packages/models/src/errors.ts

```diff
@@ -233,7 +233,7 @@ export function unauthorizedError(details: string): ApiError<CommonErrorCodes> {
 
 export function badRequestError(
   detail: string,
-  errors: Error[]
+  errors?: Error[]
```

Comment:
is there a case when we raise the `badRequestError` without attaching errors?
as far as I can tell there should always be reasons attached

Reply:
With the current API it would also become verbose, I would keep it as it is and wait for the refactor, otherwise, I would have to write:

```
      throw badRequestError("Invalid tenant attribute in API request", [
        new Error(
          `Invalid tenant attribute in API request: ${JSON.stringify(input)}`
        ),
      ]);
```